# Config for simulating sensor measurements in ET-DDF ROS framework

noise: true           # bool
depth_pub_rate: 0.00000001
usbl_pub_rate: 0.5
distance_res: 1       # m (number of decimal points)
angular_res: 1        # degrees (number of decimal points)
depth_res: 2          # m (number of decimal points)
positioning_res: 1    # m (number of decimal points)  
distance_noise: 1     # m
angular_noise: 1      # degrees
depth_noise: 0.03     # m
gps_noise: 1          # m std dev
lin_rel_noise: 1.77       # m std dev
gps_pub_rate: 5
lin_rel_pub_rate: 10

gt_topic: pose_gt # topic for ground truth data
